Fixed responsiveness & WCAG compliance

SweetlyBossy opened the pull request:
### What does this PR do?
I have now fixed the responsiveness of the code provided to me. I have also included everything needed to be WCAG compliant.
### Description of Task to be completed?
#528 #529 
### How should this be manually tested?
QA approval needed

### Any background context you want to provide?

SweetlyBossy committed:
Fixed responsiveness & WCAG compliance

I have now fixed the responsiveness of the code provided to me. I have also included everything needed to be WCAG compliant.

GustavBoe committed:
Merge branch 'develop' into 528-ticket-27-responsiveness-SaraAl

GustavBoe reviewed (changes requested):
Hi Sara! Good job on the responsiveness!
We will have to wait until all elements are in place in order for you to get a proper review. 

For now, I see that some of the links are missing an aria-label. When all the elements are in place, please take this into account when looking over your code.

SweetlyBossy committed:
Fixed the ID - Read only

As instructed the ID only read input field has now been coded to show from 768px and up. In other words not in mobile and tablet view.

SweetlyBossy committed:
Merge branch '528-ticket-27-responsiveness-SaraAl' of https://github.com/NoroffFEU/edu-gate into 528-ticket-27-responsiveness-SaraAl
